### ISSUE TITLE
Dropdown missing inverted style

### ISSUE THREAD

**Issue by raptoria** [Semantic-Org/Semantic-UI#5276]
no dark style exists for vanilla dropdown.

**Comment by Banandrew** [comment]
I spent this past week trying to figure it out, and tried various aesthetic combinations, ending up with [this](https://jsfiddle.net/1ckq2n4b/embedded/result/). I’ll use the overrides below to propose a proper PR as soon as I can, so if you’re not happy with how it turned out, now is a good time to talk about it.

@MathWhiz Thanks for patience, copy the following and rebuild the framework:

* [installation_folder/src/site/modules/dropdown.overrides](https://drive.google.com/file/d/0B4i4WJu2r-_MQThOM0pad1dpNzQ/view?usp=sharing)

* [installation_folder/src/site/modules/dropdown.variables](https://drive.google.com/file/d/0B4i4WJu2r-_MM29jdC1KYzJKbE0/view?usp=sharing)

* [installation_folder/src/site/collections/menu.overrides](https://drive.google.com/file/d/0B4i4WJu2r-_MeDY3eUdHS1F4Nms/view?usp=sharing)

* [installation_folder/src/site/collections/menu.variables](https://drive.google.com/file/d/0B4i4WJu2r-_McE1pN3ZhNjBNczA/view?usp=sharing)

**Comment by MathWhiz** [comment]
Awesome, thanks!

**Comment by JSmith2412** [comment]
Looks fantastic, I'm getting a problem - `variable @trackInvertedBackground is
   undefined`, I've checked and dropdown and menu are both set to default.

**Comment by Banandrew** [comment]
Hi @JSmith2412, could you please post a screenshot of the terminal output that you’re getting during the building process? The variable is global and should be available if you include the site component in the building process.

**Comment by JSmith2412** [comment]
Hi there - think I know why it's not working... I'm using the Meteor.JS package for Semantic UI, which hasn't been updated in some time... Some of the variables you've used don't appear in the plugin because they were only added in standard Semantic UI quite recently... I looked them up and added them but still no joy.

**Comment by JSmith2412** [comment]
Nailed it :) The non-dropping down was my fault, the menu wasn't being initialised properly (d'oh!). So there we go, for people using the Meteor JS Semantic UI plugin, you can go ahead and paste those variable definitions and it should work... Thank you @Banandrew !

**Comment by MathWhiz** [comment]
@Banandrew Have you made any progress with this?

**Comment by Banandrew** [comment]
@MathWhiz Hi, I moved to other issues in August, and ended up not opening a proper PR for it. Unfortunately, I won’t be able to work on SUI for quite a while to do it.

**Comment by MathWhiz** [comment]
Darn. Would it be okay if I submit your current changes as a PR?

**Comment by Banandrew** [comment]
@MathWhiz Of course, I'd be thankful if someone did it properly, and you didn't even need to ask.

**Comment by lubber-de** [comment]
This is available in https://fomantic-ui.com/modules/dropdown.html#inverted
